static/js/containers/ProfilePage.js

```diff
@@ -20,13 +20,19 @@ import {
 } from '../constants';
 import { createActionHelper } from '../lib/redux';
 import type { Profile } from '../flow/profileTypes';
+import { startProfileEdit } from '../actions/profile';
 
 class ProfilePage extends ProfileFormContainer {
   currentStep: Function = (): string => {
     const { ui: { profileStep } } = this.props;
     return profileStep;
   };
 
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(startProfileEdit(SETTINGS.username));
```

Comment:
Can you also change `SETTINGS.username` to `SETTINGS.user.username` here and in the test?